routes/tasks.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express')
+const router = express.Router()
+const tasksController = require('../controllers/tasksController')
+
+/**
+ * @swagger
+ * /tasks/:
+ *   summary: Used to get all the tasks
+ *   responses:
+ *     200:
+ *       description: returns tasks
+ *     404:
+ *       description : no tasks found
+ *       content:
+ *         application/json:
+ *     503:
+ *       description: serverUnavailable
+ *       content:
+ *         application/json:
```

Comment:
- for 404 we need to write 
`application/json:
        schema:
            $ref: '#/components/schemas/errors/notFound'`

- for 503
 `application/json:
         schema:
              $ref: '#/components/schemas/errors/serverUnavailable'`

Please add them wherever missing. Thank you! 😃

Reply:
Got it, Thanks.